### DISCUSSION TITLE
Standup (Friday, April 22, 2022)

### DISCUSSION THREAD

**Discussion by CarlosCRG19** [CarlosCRG19/ijalti-peers#19] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
 ```

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 -  Killed my laptop keyboard, finished writting down functional requirements #13 

**What are your priorities for the next 24 hours?**:
 -  Discuss NFR, and wirte them down in order to start working on project's mock up

**Blockers**:
 -  Define web page features 

**Shoutouts**:
 - @antonieto  @paotg123  @CristianCazares  for helping my writting down the requirements

**Reply by CristianCazares** [reply]
F

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - Contribuite on FR and NFR  (#13 ) and tried to start the introduction of the document (#12 )

**What are your priorities for the next 24 hours?**:
 - To finish the introduction xd (#12 )

**Blockers**:
 - We may need to redefine the "problem" that we initially write down. As mrs Fabiola said, we write down the problem IJALTI is trying to solve, no the one that we are trying to solve for IJALTI.

**Comment by paotg123** [comment]
**What did you achieve in the last 24 hours?**
I wrote some functional requirements

**What are your priorities for the next 24 hours?**
look for another task

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Helped @DavBot02 with #11 

**What are your priorities for the next 24 hours?**:
 - Continue with #14 

**Blockers**:
 - None

**Shoutouts**:
 - @antonieto for helping David and me with the usability requirements

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Finished #11 with @CarlosCRG19 and @antonieto 

**What are your priorities for the next 24 hours?**:
 - Continue with #12 

**Blockers**:
 -

**Shoutouts**:
 - @CarlosCRG19 @antonieto for helping to finish in order to continue with important tasks
